src/content/docs/en/guides/integrations-guide/lit.mdx

```diff
@@ -179,6 +179,8 @@ When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree
 public-hoist-pattern[]=*lit*
 ```
 
+### Astro does not support IntelliSense for Lit components
+
 ### Limitations
 
 The Lit integration is powered by `@lit-labs/ssr` which has some limitations. See their [limitations documentation](https://www.npmjs.com/package/@lit-labs/ssr#user-content-notes-and-limitations) to learn more.
```

Comment:
```suggestion
### Limitations

- The Lit integration is powered by `@lit-labs/ssr` which has some limitations. See their [limitations documentation](https://www.npmjs.com/package/@lit-labs/ssr#user-content-notes-and-limitations) to learn more.

- Astro does not support IntelliSense for Lit components.
```
Do you think this makes more sense as another "limitation" like this, maybe? We typically would not use a heading just to make a point, with no content.

Reply:
Yes, this makes a lot more sense! I've made a commit with the changes you suggested